jerry-core/parser/regexp/re-compiler.cpp

```diff
@@ -118,60 +117,60 @@ bytecode_list_insert (re_bytecode_ctx_t *bc_ctx_p, /**< RegExp bytecode context
   memcpy (src_p, bytecode_p, length);
 
   bc_ctx_p->current_p += length;
-} /* bytecode_list_insert */
+} /* re_bytecode_list_insert */
 
 /**
  * Append a RegExp opcode
  */
 static void
-append_opcode (re_bytecode_ctx_t *bc_ctx_p, /**< RegExp bytecode context */
-               re_opcode_t opcode) /**< input opcode */
+re_append_opcode (re_bytecode_ctx_t *bc_ctx_p, /**< RegExp bytecode context */
+                  re_opcode_t opcode) /**< input opcode */
 {
-  bytecode_list_append (bc_ctx_p, (re_bytecode_t*) &opcode, sizeof (re_bytecode_t));
-} /* append_opcode */
+  re_bytecode_list_append (bc_ctx_p, (re_bytecode_t*) &opcode, sizeof (re_bytecode_t));
+} /* re_append_opcode */
 
 /**
  * Append a parameter of a RegExp opcode
  */
 static void
-append_u32 (re_bytecode_ctx_t *bc_ctx_p, /**< RegExp bytecode context */
-            uint32_t value) /**< input value */
+re_append_u32 (re_bytecode_ctx_t *bc_ctx_p, /**< RegExp bytecode context */
+               uint32_t value) /**< input value */
 {
-  bytecode_list_append (bc_ctx_p, (re_bytecode_t*) &value, sizeof (uint32_t));
-} /* append_u32 */
+  re_bytecode_list_append (bc_ctx_p, (re_bytecode_t*) &value, sizeof (uint32_t));
+} /* re_append_u32 */
 
 /**
  * Append a jump offset parameter of a RegExp opcode
  */
 static void
-append_jump_offset (re_bytecode_ctx_t *bc_ctx_p, /**< RegExp bytecode context */
-                    uint32_t value) /**< input value */
+re_append_jump_offset (re_bytecode_ctx_t *bc_ctx_p, /**< RegExp bytecode context */
+                       uint32_t value) /**< input value */
 {
   value += (uint32_t) (sizeof (uint32_t));
-  append_u32 (bc_ctx_p, value);
-} /* append_jump_offset */
+  re_append_u32 (bc_ctx_p, value);
+} /* re_append_jump_offset */
 
 /**
  * Insert a RegExp opcode
  */
 static void
-insert_opcode (re_bytecode_ctx_t *bc_ctx_p, /**< RegExp bytecode context */
-               uint32_t offset, /**< distance from the start of the container */
-               re_opcode_t opcode) /**< input opcode */
+re_insert_opcode (re_bytecode_ctx_t *bc_ctx_p, /**< RegExp bytecode context */
+                  uint32_t offset, /**< distance from the start of the container */
+                  re_opcode_t opcode) /**< input opcode */
 {
-  bytecode_list_insert (bc_ctx_p, offset, (re_bytecode_t*) &opcode, sizeof (re_bytecode_t));
-} /* insert_opcode */
+  re_bytecode_list_insert (bc_ctx_p, offset, (re_bytecode_t*) &opcode, sizeof (re_bytecode_t));
+} /* re_insert_opcode */
 
 /**
  * Insert a parameter of a RegExp opcode
```

Comment:
Not exactly, but maybe that could be a solution too. What I'd like to do is to store everything based on it's value. For example if we mask the highest bit for every byte then we can store 0-127 in one byte instead of 4. Any good idea for compressing RegExp bytecode is welcome.